### PR TITLE
#666 Removed the any type from ProjectsTable.tsx

### PR DESCRIPTION
**Pull request by walker-sean**
## Changes

I removed the 'any' type from the ProjectsTable.tsx file, replaced with a created interface

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)
#666

### CODE REVIEW

File: src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx [hunk 1]
```diff
@@ -21,7 +21,13 @@ const ProjectsTable: React.FC = () => {
   const { isLoading, data, error } = useAllProjects();
   const [pageSize, setPageSize] = useState(30);
 
-  const baseColDef: any = {
+  interface BaseColDef {
```
Comment (callmejim1226) [inline src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx:24]: Great work here! Can we just have this interface located above the render name? So just put this interface above line 19.

**Review by callmejim1226 — CHANGES_REQUESTED** [review]
Great work for your first ticket! Just a small & quick comment on locating the interface. After you resolve it, I'll request Anthony to review this PR and ask him to merge it!

**Comment by walker-sean** [comment]
@callmejim1226 I've resolved the location for the interface!

**Review by Ari-S-123 — DISMISSED** [review]

**Review by anthonybernardi — COMMENTED** [review]
nice!

**Pull request merged**